contributed_definitions/NXmpes.nxdl.xml

```diff
@@ -270,6 +270,15 @@
                      group in NXsample instead.
                 </doc>
             </field>
+            <field name="atom_types" recommended="true" type="NX_CHAR">
+                <doc>
+                     Use Hill's system for listing existing elements of periodic table which are
```

Comment:
This should not be according to the Hill's system. The `chemical_formula` is according to the Hill's system and the atom_types should be a comma separated list of atom types (this at least is what happens for other appdefs). However, I think we agreed on just using `chemical_formula` and do not add `atom_types` if not present. @sanbrock is this correct?

What I think we should do:

- Do not add `atom_types`
- Make `chemical_formula` required
- Extend parser in nomad-parser-nexus to check the `chemical_formula` string to be a valid Hill's system string and to only contain valid atom types.

Reply:
for the moment we can add the field atom types, but please note that the intention is to populate its NOMAD counterpart automatically from chemical formula. So,  
- if chem formula is know, write it, otherwise  
- list elements in atom types

Reply:
@domna doc in 'atom_types' has been modified.